src/baseClass.js

```diff
@@ -1,5 +1,6 @@
 import { topSection } from './topSection.js'
```

Comment:
Continent name wise sorting doesn't seem to be clickable now, could you fix that?